source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Enhance import-export with OVA
+category: feature
+authors: arik
+wiki_category: Feature
+wiki_title: Features/Enhance import-export with OVA
+wiki_revision_count: 1
+wiki_last_updated: 2017-03-20
+feature_name: Enhance import-export with OVA
+feature_modules: engine, vdsm
+feature_status: planning
+---
+
+# Backgroud
+Open Virtual Appliance (OVA) is a package that contains the different elements of a Virual Machine (VM). OVA is typically a tar archive file that includes the configuration of the VM in the form of the Open Virtualization Format (OVF) as well as virtual disks and other VM-related data.  
+
+OVA is a known format that is used by different organizations and virtualization platforms. Its specification can be found [here](http://www.dmtf.org/standards/ovf). The standard aims at being an ["open, secure, portable, efficient and extensible format for the packaging and distribution of software to be run in virtual machines"](https://en.wikipedia.org/wiki/Open_Virtualization_Format).  
+
+Recently, oVirt was enhanced with support for importing OVAs that were generated by VMware. In addition, oVirt makes an extensive use of the OVF standard to describe VM configuration in, e.g., VM snapshots and exported VMs.  
```

Comment:
Suggested edits:
1. oVirt also makes extensive use of the OVF standard to describe VM configurations, such as VM snapshots and exported VMs.